clickhouse-http-client/src/main/java/com/clickhouse/client/http/ClickHouseHttpConnectionFactory.java

```diff
@@ -21,6 +25,7 @@ public static ClickHouseHttpConnection createConnection(ClickHouseNode server, C
         } catch (IOException e) {
             throw e;
         } catch (Throwable t) {
+            log.warn("Error when creating http client " + provider.name() + ", will use HTTP_URL_CONNECTION", t);
```

Comment:
Can we change to below? And similarly for the same class under java11 directory.
```java
log.warn("Error when creating http client %s, will use HTTP_URL_CONNECTION", provider.name(), t);
```